src/univariate/continuous/logitnormal.jl

```diff
@@ -0,0 +1,153 @@
+#using Optim # numerical estimation routines moved to NormalTransforms
+
+"""
+    LogitNormal(μ,σ)
+
+The *logit normal distribution* is the distribution of the logistic function 
```

Comment:
bad wording, it is not the distribution of a function `f` but the distribution of a transformed variable `f(X)`